js/app.js

```diff
@@ -0,0 +1,48 @@
+$(function(){
+
+  //listen for a click even on submit and do the function handleSubmit
+  $("#submitBtn").click(handleSubmit);
+
+  //city equals field input, pull the input from field and store it as var city
+  function handleSubmit(event) {
+    event.preventDefault();
+    var city = $("#cityType").val();
+    console.log("city", city)
+  }
+
+  //listen for a click even on submit and do the function changeBackground
+  $("#submitBtn").click(changeBackground);
+
+  //if input is "New York" or "New York City" or "NYC" change the background of the page nyc.jpg
+  //if input is "San Francisco" or "SF" or "Bay Area" change the background of the page sf.jpg
+  //if input is "Los Angeles" or "LA" or "LAX" change the background of the page la.jpg
+  //if input is "Austin" or "ATX" change the background of the page austin.jpg
+  //if input is "Sydney" or "SYD" change the background of the page sydney.jpg
+  //if anything else display default
+  function changeBackground(event) {
+    event.preventDefault();
+    var city = $("#cityType").val();
+
+    if (city === "New York" || "New York City" || "NYC") {
```

Comment:
@sofiaayala this should be ` if (city === "New York" || city === "New York City" || city === "NYC") `. Remember in your conditional you are asking if city equals `New York` or if city equals `New York City` or if city equals `NYC`. Try applying the same logic though out you code. 🔨